src/components/List.js

```diff
@@ -55,8 +118,10 @@ const List = ({ token }) => {
                 id={doc.id}
                 onChange={handleCheckbox}
                 checked={checkPurchasedDate(doc.lastPurchased)}
+                //disabled={checkPurchasedDate(doc.lastPurchased)} // Commented out for PR purposes only
               />
               {doc.itemName}
+              <p>Time until next purchase {doc.durationOfTime}</p>
```

Comment:
It appears this value is missing when I QA tested on netlify:
![](https://cdn.zappy.app/8bce34fd25fe494a3711fc521877e6cb.png)

Reply:
@chrisdopuch The above issue has been fixed. Thanks for hopping on Zoom with me earlier. Things seem to be working 🤞🏽 😅

Reply:
Thanks for taking care of that! Also, @chrisdopuch , those are some great list items.

Reply:
Everyone needs 🧦 🧦  and gummie bears!